tox.ini

```diff
@@ -16,8 +16,8 @@ extras =
 deps =
     anyio>=3.6.1
     asyncclick>=8.1.3.4
-    cairo-lang==0.10.0
-    starknet-devnet==0.3.1
+    cairo-lang==0.10.1
```

Comment:
If we have to do nothing to support 0.10.2, why not support 0.10.2?

Reply:
this is just for internal tests, the package will support 0.10.2 immediately even with this. it's the same, no strong opinions on bumping it.

Reply:
That's true for tox.ini, but to support 0.10.2 immediately, we need to bump the version in `setup.cfg` to 0.10.2 instead of 0.10.1, right? If not installing nile would bring 0.10.1, not 0.10.2. We are using low-level functions from cairo-lang now though, no sure if something will break from its internal API.

Reply:
I agree with Eric's comment in keeping the setup.cfg and tox.ini consistent. That said, I just tried installing cairo-lang v0.10.2 and I'm on 15 minutes so far with pip checking version compatibility. Here's a small glimpse:

```
Using cached certifi-2022.5.18.1-py3-none-any.whl (155 kB)
  Using cached certifi-2021.10.8-py2.py3-none-any.whl (149 kB)
INFO: pip is looking at multiple versions of certifi to determine which version is compatible with other requirements. This could take a while.
  Using cached certifi-2021.5.30-py2.py3-none-any.whl (145 kB)
  Using cached certifi-2020.12.5-py2.py3-none-any.whl (147 kB)
  Using cached certifi-2020.11.8-py2.py3-none-any.whl (155 kB)
  Using cached certifi-2020.6.20-py2.py3-none-any.whl (156 kB)
  Using cached certifi-2020.4.5.2-py2.py3-none-any.whl (157 kB)
INFO: This is taking longer than usual. You might need to provide the dependency resolver with stricter constraints to reduce runtime. If you want to abort this run, you can press Ctrl + C to do so. To improve how pip performs, tell us what happened here: https://pip.pypa.io/surveys/backtracking
  Using cached certifi-2020.4.5.1-py2.py3-none-any.whl (157 kB)
  Using cached certifi-2020.4.5-py2.py3-none-any.whl (156 kB)
  Using cached certifi-2019.11.28-py2.py3-none-any.whl (156 kB)
  Using cached certifi-2019.9.11-py2.py3-none-any.whl (154 kB)
  Using cached certifi-2019.6.16-py2.py3-none-any.whl (157 kB)
  Using cached certifi-2019.3.9-py2.py3-none-any.whl (158 kB)
  Using cached certifi-2018.11.29-py2.py3-none-any.whl (154 kB)
  Using cached certifi-2018.10.15-py2.py3-none-any.whl (146 kB)
  Using cached certifi-2018.8.24-py2.py3-none-any.whl (147 kB)
  Using cached certifi-2018.8.13-py2.py3-none-any.whl (146 kB)
  Using cached certifi-2018.4.16-py2.py3-none-any.whl (150 kB)
  Using cached certifi-2018.1.18-py2.py3-none-any.whl (151 kB)
  Using cached certifi-2017.11.5-py2.py3-none-any.whl (330 kB)
  Using cached certifi-2017.7.27.1-py2.py3-none-any.whl (349 kB)
  Using cached certifi-2017.7.27-py2.py3-none-any.whl (349 kB)
  Using cached certifi-2017.4.17-py2.py3-none-any.whl (375 kB)
INFO: pip is looking at multiple versions of requests to determine which version is compatible with other requirements. This could take a while.
Collecting requests>=2.28.1
```

I'll update when this process finishes, but I suggest we keep v0.10.1 for now